components/Content/Article.jsx

```diff
@@ -21,27 +21,23 @@ const Article = ({ article, href, showReadTime, showDate }) => {
     return data?.attributes?.formats?.large;
   }, [article]);
 
-  const {
-    title,
-    publishedAt: unFormattedDatePublished,
-    readTime,
-  } = article.attributes;
+  const { title, datePublished, readTime } = article.attributes;
```

Comment:
I remember this was breaking at some page (don't remember which one exactly), which is why I made the change 🤔 
by any chance do you know which page is [this](https://github.com/valory-xyz/olas-website/pull/314) (the ss in the PR)